internal/integration_test/vs/bench_allocation_cgo.go

```diff
@@ -0,0 +1,89 @@
+package vs
```

Comment:
thanks, but no thanks on this, as these benchmarks run on every change. If we are going to spend on this it should be a notably different wasm use case, not a slight variant of an existing one.

Reply:
done